### ISSUE TITLE
Mumble URL linking still doesn't work.

### ISSUE THREAD

**Issue by crius-al** [mumble-voip/mumble#5902]
### Description

https://github.com/mumble-voip/mumble/issues/5473#issuecomment-1018438938%C2%A0 describes my problem perfectly.

https://github.com/mumble-voip/mumble/issues/5534 seems to have been closed too soon.

### Steps to reproduce

Install mumble, click on a URL link.

### Mumble version

1.4.287

### Mumble component

Client

### OS

Windows

### Reproducible?

Yes

### Additional information

[Window Title]
mumble://private mumble server

[Content]
This file does not have an app associated with it for performing this action. Please install an app or, if one is already installed, create an association in the Default Apps Settings page.

[OK]

**Comment by davidebeatrici** [comment]
#5521 and #5528 are the relevant PRs; are the registry keys created in your case?

**Comment by crius-al** [comment]
> #5521 and #5528 are the relevant PRs; are the registry keys created in your case?

The registry entries were not created in my case.

**Comment by Krzmbrzl** [comment]
Have you tried first completely uninstalling Mumble and then re-installing it? Maybe it only adds the registry keys on the initial installation? :thinking:

**Comment by JVKeller** [comment]
I would like to see this working in android. I have zebra scanners that I am going to install Mumla on, and if a user could just scan a barcode with the link (mumble://) that would save a lot of setup!

**Comment by crius-al** [comment]
> Have you tried first completely uninstalling Mumble and then re-installing it? Maybe it only adds the registry keys on the initial installation? 🤔

I tried completely uninstalling mumble and reinstalling it three times now, and it made no difference.
Uninstalling mumble 1.34 removed the keys. Installing 1.4.287 did not enter the keys into the registry.

**Comment by crius-al** [comment]
Am I the only one experiencing this problem? Have other people tested with the newest mumble and NOT gotten the problem in Windows 10?

**Comment by Krzmbrzl** [comment]
@JVKeller we don't do Android. If you have any requests there, you'll have to submit it to the respective project (Mumla).

@crius-al hm okay. I thought it has been tested to work, but I'm not entirely sure. And since none of the main developers is using Windows, it could easily be that this slipped past us so far...

**Comment by BrianOlesen02** [comment]
Yes I try to install Mumble for the first time, and get this responce in Danish that i hope you can google translate yourself. Guiden Mumble (client) Installation blev afsluttet før tid på grund af en fejl. Systemet er ikke blevet ændret. Hvis du vil installere programmet på et senere tidspunkt, skal du køre guiden Installation igen. Klik på knappen Udfør for at afslutte guiden Installation. Installationsprogrammet til Mumble (client) blev afsluttet før tid 
Udfør Knap" Somthing about was cancelled early due to an error. So I'm not able to install Mumble on a Windows 10 pro machine. Hope this will be resolved in not too long.

**Comment by Krzmbrzl** [comment]
@BrianOlesen02 how does this relate to the issue at hand?

**Comment by BrianOlesen02** [comment]
Hi,

Well wasn’t it about problems with the installer?

 

Fra: Robert Adam ***@***.***> 
Sendt: 8. november 2022 21:17
Til: mumble-voip/mumble ***@***.***>
Cc: BrianOlesen02 ***@***.***>; Mention ***@***.***>
Emne: Re: [mumble-voip/mumble] Mumble URL linking still doesn't work. (Issue #5902)

 

@BrianOlesen02 <https://github.com/BrianOlesen02>  how does this relate to the issue at hand?

**Comment by Krzmbrzl** [comment]
Not general problems, no. This issue is specifically about the installer not creating the registry entries needed for Mumble URLs to work correctly.

**Comment by BrianOlesen02** [comment]
HI,

Ok.

I got it to work by installing the packages and choosing install from the context menu.

 

Brian

Brian

 

Fra: Robert Adam ***@***.***> 
Sendt: 9. november 2022 07:25
Til: mumble-voip/mumble ***@***.***>
Cc: BrianOlesen02 ***@***.***>; Mention ***@***.***>
Emne: Re: [mumble-voip/mumble] Mumble URL linking still doesn't work. (Issue #5902)

 

Not general problems, no. This issue is specifically about the installer not creating the registry entries needed for Mumble URLs to work correctly.
